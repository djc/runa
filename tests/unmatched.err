unmatched.rns [2.13]: value of type 'anyint' may only be compared to integer type
    print(1 == 'a')
            ^
