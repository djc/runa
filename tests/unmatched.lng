def main(name: str, args: array[str]) -> int:
	print(1 == 'a')
