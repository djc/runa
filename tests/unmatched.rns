def main():
	print(1 == 'a')
