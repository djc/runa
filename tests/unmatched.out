tests/unmatched.lng [2.10]: unmatched types 'int', 'str'
    print(1 == 'a')
            ^
